src/M6Web/Bundle/ElasticsearchBundle/Connection/CurlMultiConnectionDecorator.php

```diff
@@ -45,11 +45,14 @@ public function performRequest($method, $uri, $params = null, $body = null, $opt
             throw $e;
         }
 
+        $took = $this->extractTookFromResponse($response);
+
         $this->dispatchEvent(
             $method,
             $uri,
-            $response['requestInfo']['http_code'],
-            $response['requestInfo']['total_time'] * 1000  // Convert from seconds to milliseconds
+            $response['status'],
```

Comment:
`CurlMultiConnection` returns an array with keys `status`, `text`, `info` ([see](https://github.com/elastic/elasticsearch-php/blob/master/src/Elasticsearch/Connections/CurlMultiConnection.php#L243-L247)).

All connections classes in elasticsearch-php return the same array format.

I don't think we ever used/tested `CurlMultiConnectionDecorator`.